solr/core/src/test/org/apache/solr/cloud/ReplaceNodeTest.java

```diff
@@ -158,12 +165,7 @@ public void test() throws Exception {
     // make sure all newly created replicas on node are active
     List<Replica> newReplicas = collection.getReplicas(node2bdecommissioned);
     replicas.forEach(r -> {
```

Comment:
Curly brackets are not needed:

`replicas.forEach(r -> newReplicas.removeIf(nr -> nr.getName().equals(r.getName())));`